hutch_python/cli.py

```diff
@@ -98,17 +99,15 @@ def configure_ipython_session():
     # Set up tab completion modifications
     configure_tab_completion(ipy_config)
 
-    # remove force exit key bind from <ctrl-\\>
-    ipy_config.InteractiveShellApp.exec_lines = (
-        "import IPython; "
-        "from prompt_toolkit.keys import Keys; "
-        "ip = IPython.get_ipython(); "
-        "cb = Keys.ControlBackslash; "
-        "ip.pt_app.key_bindings.remove(cb) "
-    )
-
-    # add env info to ipython banner
-    ipy_config.TerminalInteractiveShell.banner2 = get_env_info()
+    # disable default banner
+    ipy_config.TerminalIPythonApp.display_banner = False
+
+    # Run startup hook code, print banner after startup hook files
+    files = [
+             str(Path(__file__).parent / 'startup_script.py'),
```

Comment:
We apparently have a constant for this path already -
```suggestion
             str(DIR_MODULE / 'startup_script.py'),
```